ChangeLog.d/add-missing-parenthesis.txt

```diff
@@ -0,0 +1,4 @@
+Bugfix
+   * Add a parenthesis that was missing from ecp.c when
```

Comment:
Sorry for being picky, but ChangeLog entries should describe what is fixed (the negative behaviour observable by users) rather than, or at least before, how it's fixed or what the cause was. So that would be "Fix a compilation error when MBEDTLS_ECP_RANDOMIZE_MXZ_ALT is defined." If you want, you can add that is was caused by unbalanced parentheses but I'm not sure it adds much value.